### PR TITLE
Fix: Correct poor grammar and add ways PO supports SM

### PR DESCRIPTION
**Pull request by jdmedlock**

### CODE REVIEW

**Comment by zizi1404** [comment]
looks good

**Review by zizi1404 — COMMENTED** [review]
looks good.

File: docs/guides/voyage/voyage.md [hunk 1]
```diff
@@ -82,16 +82,25 @@ Voyages have specific requirements, milestones, and ***require a commitment of 8
         <li>Ensuring that the Product Backlog is transparent, visible and understood</li>
         <li>Making sure all stakeholders are represented and have a "voice" in the project</li>
       </ul>
+      The PO also works closely with the Scrum Master by:
+      <ul>
+        <li>Ensuring that any changes to the backlog from other stakeholders,
```
Comment (christopher-maxwell) [inline docs/guides/voyage/voyage.md:87]: I like the idea here. The updates to backlog from stakeholders is a good suggestion and practical, but transparency should already exist, and it's not common to have users updating the backlog. We could address that by changing 'users' to 'developers', but I wonder if we need this emphasized. I common problem with teams is a disinterested PO who doesn't attend ceremonies, so I wonder about highlighting these four collaborative points for how the Production Owner works closely with the Scrum Master?

        <li>Participating in Sprint Planning, Review, and Retrospective meetings</li>
        <li>Actively listening to all ideas and concerns from the team</li>
        <li>Quickly resolving any issues or suggestions related to the backlog</li>
        <li>Working closely together to ensure that the needs of all stakeholders, 
        including the development team, are met</li>

**Review by christopher-maxwell — COMMENTED** [review]
Hi Jim, looks good... I had a few thoughts on the PO section.

File: docs/guides/voyage/voyage.md (hunk 1 above)
Comment (jdmedlock) [reply docs/guides/voyage/voyage.md]: @christopher-maxwell I like this a lot! I've update this with your suggestion. Thank you

**Comment by jdmedlock** [comment]
> looks good.

Thank you!

**Review by christopher-maxwell — COMMENTED** [review]
Looks good!

File: docs/guides/voyage/voyage.md (hunk 1 above)
Comment (christopher-maxwell) [reply docs/guides/voyage/voyage.md]: Great! looks good

**Pull request merged**